lib/src/core/util/numbers_constants.dart

```diff
@@ -1,16 +1,31 @@
 class NumbersConstants {
   static const double paddingLeftSmall = 5.0;
-  static const double paddingLeftMedium = 10.0;
-  static const double paddingLeftBig = 30.0;
+  static const double paddingLeftXSmall = 10.0;
+  static const double paddingLeftMedium = 20.0;
+  static const double paddingLeftBig = 25.0;
+  static const double paddingLeftXBig = 30.0;
+
   static const double paddingTopSmall = 5.0;
-  static const double paddingTopMedium = 8.0;
-  static const double paddingTopBig = 20.0;
+  static const double paddingTopXSmall = 8.0;
+  static const double paddingTopXXSmall = 10.0;
+  static const double paddingTopMedium = 15.0;
+  static const double paddingTopXMedium = 20.0;
+  static const double paddingTopLarge = 60.0;
+
   static const double paddingRightSmall = 8.0;
-  static const double paddingRightMedium = 10.0;
+  static const double paddingRightXSmall = 10.0;
+  static const double paddingRightMedium = 20.0;
   static const double paddingRightBig = 30.0;
-  static const double paddingBottomSmall = 10.0;
-  static const double paddingBottomMedium = 20.0;
+  static const double paddingRightLarge = 35.0;
+
+  static const double paddingBottomSmall = 7.0;
+  static const double paddingBottomXSmall = 10.0;
+  static const double paddingBottomMedium = 15.0;
+  static const double paddingBottomXMedium = 20.0;
   static const double paddingBottomBig = 30.0;
```

Comment:
As an approach, you can standardize padding into different dimensions like, paddingSmall, paddingXSmall, paddingMedium, etc.